libbpf-tools/readahead.bpf.c

```diff
@@ -24,7 +24,7 @@ struct {
 	__uint(map_flags, BPF_F_NO_PREALLOC);
 } birth SEC(".maps");
 
-static struct hist hist;
+struct hist hist;
```

Comment:
probably want to initialize explicitly to avoid problems with slightly older Clangs

Reply:
Make sense. I'll update the MR.